src/app/pages/subjects/subjects.js

```diff
@@ -1,114 +1,116 @@
-        };
-        window.addEventListener('popstate', this.filterSubjectsEvent);
-        this.regions.filter.attach(this.categorySelector);
-        const category = this.categoryFromPath();
+    function setCategory(newCategory) {
+        const categoryEntry = categories.find((e) => e.value === newCategory);
+        const path = newCategory === 'view-all' ? pagePath : `${pagePath}/${newCategory}`;
 
-        this.categorySelector.updateSelected(category);
-        this.filterSubjects(category);
-        this.el.classList.add('loaded');
+        routerBus.emit('navigate', path, {
+            filter: newCategory,
+            path: pagePath
+        });
+        document.title = (categoryEntry && 'title' in categoryEntry) ? categoryEntry.title : model.title;
     }
 
-    onLoaded() {
-        const threshold = 347;
+    const {
+        page_description: heroHtml,
+        dev_standard_1_heading: blurbHeading1,
+        dev_standard_1_description: blurbDescription1,
+        dev_standard_2_heading: blurbHeading2,
+        dev_standard_2_description: blurbDescription2,
+        dev_standard_3_heading: blurbHeading3,
+        dev_standard_3_description: blurbDescription3,
+        books
+    } = model;
+
+    return (
+        <React.Fragment>
+            <RawHTML className="hero" html={heroHtml} />
+
+            <div className="strips-and-filter">
+                <img className="strips" src="/images/components/strips.svg" height="10" alt="" role="presentation" />
+                <div className={`filter ${model.filterIsSticky ? 'sticky': ''}`}>
+                    <RadioPanel2 selectedItem={category} items={categories} onChange={setCategory} />
+                </div>
+            </div>
+            <div className="books">
+                <div className="boxed container">
+                    <BookViewer books={books} category={category} />
+                </div>
+                <div>
+                    <h2 className="text-content">About Our Textbooks</h2>
+                    {
+                        blurbHeading1 &&
+                            <div className="boxed-row feature-block">
+                                <div className="blurb">
+                                    <h3 className="title">{blurbHeading1}</h3>
+                                    <RawHTML Tag="p" className="text" html={blurbDescription1} />
+                                </div>
+                                <div className="blurb">
+                                    <h3 className="title">{model.dev_standard_2_heading}</h3>
+                                    <RawHTML Tag="p" className="text" html={blurbDescription2} />
+                                </div>
+                                <div className="blurb">
+                                    <h3 className="title">{model.dev_standard_2_heading}</h3>
+                                    <RawHTML Tag="p" className="text" html={blurbDescription3} />
+                                </div>
```

Comment:
Something that I like to do with common things like the `RawHTML` with a props that's used often is to export a small wrapper that sets that prop.  For instance you could make a `P`:
```js
const P = (props) => <RawHTML Tag="p" {...props}
```
Then you could render `<P  className="text" html={blurbDescription3} />` here